dune_api_scripts/duneanalytics.py

```diff
@@ -0,0 +1,210 @@
+# Code mostly copied from: https://github.com/itzmestar/duneanalytics
+
+# -*- coding: utf-8 -*- #
```

Comment:
Nit - This is usually at the very top of the file. Also, these coding headers haven't been necessary since python3.

cf - https://stackoverflow.com/a/58652741/5601507

```suggestion
```